src/dodal/devices/smargon.py

```diff
@@ -116,7 +116,7 @@ class CombinedMove(TypedDict):
     chi: NotRequired[float | None]
 
 
-class Smargon(StandardReadable, Movable):
+class Smargon(XYZStage, Movable):
```

Comment:
Not for self really, this should be a `SixAxisGonio`, we can fix this up at a later date though